From d3c44f73a61d44e53612fde282466a3479f33739 Mon Sep 17 00:00:00 2001
From: Amos Kong <akong@redhat.com>
Date: Thu, 7 Nov 2013 08:38:28 +0100
Subject: [PATCH 25/25] don't disable ctrl_mac_addr feature for 6.5 machine (RHEL only)

RH-Author: Amos Kong <akong@redhat.com>
Message-id: <1383813508-6353-1-git-send-email-akong@redhat.com>
Patchwork-id: 55592
O-Subject: [RHEL-7.0 qemu-kvm PATCH] don't disable ctrl_mac_addr feature for 6.5 machine (RHEL only)
Bugzilla: 1005039
RH-Acked-by: Orit Wasserman <owasserm@redhat.com>
RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
RH-Acked-by: Stefan Hajnoczi <stefanha@redhat.com>

Bugzilla: 1005039
Brew: http://brewweb.devel.redhat.com/brew/taskinfo?taskID=6540840
Test: tested by myself

ctrl_mac_addr is already supported in qemu-kvm 6.5,
we should not disable this feature for 6.5 machine type.

Signed-off-by: Amos Kong <akong@redhat.com>
---
 hw/i386/pc_piix.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/i386/pc_piix.c |    8 ++++----
 1 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/hw/i386/pc_piix.c b/hw/i386/pc_piix.c
index 2d8e79f..8bae4c5 100644
--- a/hw/i386/pc_piix.c
+++ b/hw/i386/pc_piix.c
@@ -784,10 +784,6 @@ static QEMUMachine pc_machine_rhel700 = {
         .value    = stringify(1),\
     },{\
         .driver   = "virtio-net-pci",\
-        .property = "ctrl_mac_addr",\
-        .value    = "off",\
-    },{\
-        .driver   = "virtio-net-pci",\
         .property = "mq",\
         .value    = "off",\
     },{\
@@ -916,6 +912,10 @@ static QEMUMachine pc_machine_rhel650 = {
         .driver   = "hda-output",\
         .property = "mixer",\
         .value    = "off",\
+    },{\
+        .driver   = "virtio-net-pci",\
+        .property = "ctrl_mac_addr",\
+        .value    = "off",\
     }
 
 static void pc_init_rhel640(QEMUMachineInitArgs *args)
-- 
1.7.1

